api/create_hypertable.md

```diff
@@ -17,22 +17,22 @@ still work on the resulting hypertable.
 |Name|Type|Description|
 |---|---|---|
 | `relation` | REGCLASS | Identifier of table to convert to hypertable. |
-| `time_column_name` | REGCLASS | Name of the column containing time values as well as the primary column to partition by. |
+| `time_column_name` | NAME | Name of the column containing time values as well as the primary column to partition by. |
```

Comment:
Please compress/unspace these tables.

Reply:
Spun off to #908 to avoid cluttering review on this PR